Bunny

erik-stodola opened the pull request:
Removed UFO, Added 2x Bunnies, Fixed Comments/Code, Fixed Kyle Code, Added Normal Support for Objects (Help you out Rob)

kyle-ilantzis commented:
You still have UFO model and other UFO related things? Do you mean to keep them?
We have the disco ball, great! But will you remove the sphere from the obstacles?
The bunny stuff is cool, good job.

To sum up: remove old UFO stuff, remove sphere from obstacles.

erik-stodola commented:
Removed redundant UFO code, and fixed the obstacles. I am not sure how at the moment to incorporate lazy loading in model importing so I will focus on other tasks at the moment.
